### ISSUE TITLE
Local deps?

### ISSUE THREAD

**Issue by den1k** [nextjournal/clerk#30]
I just painfully found out on a flight that clerk's frontend deps (in my case vega, vega lite, vega embed and katex) are not included in the source and therefore failed fetching in-browser. This is an edge case for sure, but since clerk advertises itself being local first, I think it would be nice for clerk to be able to work offline.

**Comment by mk** [comment]
True, this is annoying and Clerk should support working completely offline. I'll revisit this after #36 which switches to [tailwind's play cdn](https://tailwindcss.com/blog/tailwindcss-v3#play-cdn) and is another thing to consider for offline support.

**Comment by borkdude** [comment]
Working on this.

**Comment by borkdude** [comment]
This has now been addressed in PR #113.
Before we merge this, please test commit ccae890217c56a6826d6833af3503e1c52a575b6. 

You can cache assets locally, before entering a plane, with:

```
clojure -X nextjournal.clerk/cache-assets!
```

But also while using clerk, assets will be cached on first use.

**Comment by borkdude** [comment]
Anyone? :)

**Comment by mk** [comment]
Was chatting with @borkdude about resuming work on this. When we actually tried it we discovered to my surprise that Clerk does work fine when offline, as long as you previously had a notebook with the viewers you're using open. This includes KaTeX, Vega, Plotly etc which are all cached in the browser and do survive a reload. So this is a lot better than how I thought it worked.

We might want to consider bundling these deps in the jar in the future but would track that in a separate issue when the need arises.
